artemis/metrics.py

```diff
@@ -0,0 +1,65 @@
+import time
+from typing import Generator
+
+from karton.core.backend import KartonBackend, KartonMetrics
+from karton.core.config import Config as KartonConfig
+from prometheus_client import (
+    GC_COLLECTOR,
+    PLATFORM_COLLECTOR,
+    PROCESS_COLLECTOR,
+    REGISTRY,
+    start_http_server,
+)
+from prometheus_client.core import GaugeMetricFamily
+from prometheus_client.registry import Collector
+
+
+class ArtemisMetricsCollector(Collector):
```

Comment:
I think values provided in this Collector should have been (or should include) sums over period of time (eg. day or scan cycle)

Reply:
IMO nope - this endpoint includes current status and calculating other statistics out of that (e.g. daily change) is the resposibility of e.g. the tool that plots the statistics (e.g. Grafana)